vrtool_docs/Gebruikershandleiding/Preprocessing/werken_met_preprocessor.rst

```diff
@@ -0,0 +1,69 @@
+================================
+Werken met de preprocessor
+================================
+
+Opstarten van de preprocessor
+=============================
+
+Alle workflows (de blauwe rechthoeken in de overzichtsfiguur) gebruiken dezelfde stappen. Om workflows uit te voeren moet eerst Anaconda Prompt worden opgestart en het environment worden geactiveerd.
+Het activeren van een environment kan door het volgende command in Anaconda in te voeren. 
+
+.. code-block:: bash
+
+    conda activate C:/Veiligheidsrendement/.env/
+
+.. note::
+    **Let op:**
+    Vervang ``"C:/Veiligheidsrendement"`` met de locatie van de map waar de verschillende bestanden zijn neergezet.
+    
+    Als je het environment een andere naam hebt gegeven, vervang ``".env"`` dan door de naam die je aan het environment gegeven hebt.
+
+
+Aanmaken van een mappenstructuur
+================================
+
+De tweede stap is om een werkmap te maken voor de preprocessor. Dat gebeurt met een aparte workflow ``maak_project``. Hierin wordt automatisch een complete mappenstructuur aangemaakt waarmee de preprocessor uit de voeten kan. Dit is te doen via het volgende commando:
+
+.. code-block:: bash
+
+    python -m preprocessing maak_project --project_folder {projectfolder} --traject_id {traject_id}
+
+Bij ``{projectfolder}`` moet de locatie van de map worden ingevuld waar de mappenstructuur moet worden aangemaakt, bijvoorbeeld ``"C:/Veiligheidsrendement/preprocessor_werkmap"``. Bij ``{traject_id}`` moet het traject worden ingevuld, bijvoorbeeld ``"24-3"``.
+
+.. note::
+    **Tip** 
+    Voor de verschillende workflows kan met ``python -m preprocessing workflow_naam --help`` een overzicht van de invoerparameters worden opgevraagd.
+
+De mappenstructuur die wordt aangemaakt ziet er als volgt uit:
+
+.. figure:: img/Mappenstructuur.png
+   :alt: Mappenstructuur
+   :align: center
+
+   Mappenstructuur t.b.v. de preprocessor
+
+De structuur van de werkmap
+___________________________
+
+In de map ``input_files`` zijn 5 submappen aangemaakt:
+
+* ``bag_gebouwen``: hierin kan een geopackage van de Basisadministratie Adressen en Gebouwen (BAG) worden geplaatst. Deze is te downloaden vanaf de website van `PDOK<https://service.pdok.nl/lv/bag/atom/downloads/bag-light.gpkg>`_. Let op: dit is een zeer groot bestand (ca. 7 GB), dus het kan even duren.
```

Comment:
Hier toevoegen dat je de BAG niet in elke projectfolder hoeft te zetten, maar gewoon ergens centraal. Dan moet je wel in je config het pad aanpassen naar de BAG. Dat mag dan een absoluut pad zin